packages/block-editor/src/components/block-settings-menu-controls/index.js

```diff
@@ -31,9 +36,14 @@ const BlockSettingsMenuControlsSlot = ( { fillProps, clientIds = null } ) => {
 
 	return (
 		<Slot fillProps={ { ...fillProps, selectedBlocks } }>
-			{ ( fills ) =>
-				! isEmpty( fills ) && <MenuGroup>{ fills }</MenuGroup>
-			}
+			{ ( fills ) => {
+				return (
+					<MenuGroup>
+						{ ! isEmpty( fills ) && fills }
+						<ConvertToGroupButton />
```

Comment:
Do we want to have this option always displayed as the last one? There might be several custom items injected by plugin authors. We should consider having it before `fills`, but it would also mean that it would flip the order with reusable blocks.

Reply:
That's why I placed it below, because of the `reusable` order.. I don't have a strong opinion on this and don't have any context if the order was discussed in the past